### ISSUE TITLE
Logbook,WPs,Inventory not shown

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#1055]
Since NB from 26.01. it appeared to me, that the logbook-page of stored cache are no longer available.
Only after updating the cache, its available again but after a while (or restart of the app, or some other action) the logbook is again no longer shown in cache-details.

Right now I can find a way to reproduce it but I am quite sure its a new issue in the nightly from yesterday (26.01.).
